docs/plugins/bpsync.rst

```diff
@@ -0,0 +1,37 @@
+BPSync Plugin
+=============
+
+This plugin provides the ``bpsync`` command, which lets you fetch metadata
+from Beatport for albums and tracks that already have Beatport IDs. This
+is useful for updating tags as they are fixed in the Beatport database, or
+when you change your mind about some config options that change how tags are
+written to files. If you have a music library that is already nicely tagged by
+a program that also uses Beatport, this can speed up the initial import if you
+just import "as-is" and then use ``bpsync`` to get up-to-date tags that are written
+to the files according to your beets configuration.
```

Comment:
This last argument is somewhat less relevant for Beatport than it is for MusicBrainz. There are lots of other MB taggers out there (e.g., Picard), but I don't know of any other Beatport-based metadata taggers. Maybe it should reference that you might have downloaded your music from Beatport in the first place?